webpack.config.js

```diff
@@ -0,0 +1,29 @@
+var Encore = require('@symfony/webpack-encore');
+
+Encore
+    .setOutputPath('web/build/')
+    .setPublicPath('/build')
+
+    .copyFiles({
+        from: './web/images/',
+        to: 'images/[path][name].[hash:8].[ext]',
+    })
+
+    .addEntry('app', './web/js/app.js')
+    .addEntry('my-connections', './web/js/my-connections.js')
+
+    // will require an extra script tag for runtime.js
+    // but, you probably want this, unless you're building a single-page app
+    .disableSingleRuntimeChunk()
```

Comment:
Is there a reason to disable the single runtime chunk?

Reply:
Yes, it prevents jquery from being loaded globally, and I was not able to fix this in an elegant manner. This was the proposed fix I read on a Symfony issue.